device/iot-device-client/src/main/java/com/microsoft/azure/sdk/iot/device/DeviceClientConfig.java

```diff
@@ -197,46 +201,19 @@ public DeviceClientConfig(IotHubConnectionString iotHubConnectionString, ClientO
         }
     }
 
+    @Builder
```

Comment:
What does this annotation do when added to a constructor rather than to the class?

Reply:
James can correct me if I am wrong but I believe a class with no constructors is given the annotation at class level, while if you have a constructor defined you'll need to add the annotation to the constructor directly

Reply:
I don't see the other constructors in the class being given this annotation, anything special about this particular constructor?